VideoLocker/src/main/java/org/edx/mobile/module/notification/UserNotificationManager.java

```diff
@@ -0,0 +1,169 @@
+package org.edx.mobile.module.notification;
+
+import com.parse.ParseException;
+import com.parse.ParsePush;
+import com.parse.SaveCallback;
+
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.model.api.CourseEntry;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.module.prefs.UserBasedPrefManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * we need to keep track of which course is unsubscribed as Parse
+ * only return the courses we currently subscribed.
+ *
+ * The design concerns - TODO -
+ * should we keep the record of the subscriptions? if we fetch it from
+ * parse server, what's the frequency of fetching?
+ * if we cache it locally, when to do the resync with parse server?
+ *
+ */
+public class UserNotificationManager {
+    private static final Logger logger = new Logger(UserNotificationManager.class.getName());
+
+    public static final UserNotificationManager instance = new UserNotificationManager();
+
+    private UserNotificationManager(){
+
+    }
+
+    /**
+     * if use log out, he should not receive the push notification
```

Comment:
Maybe make this "if a user logs out, they should not receive the push notification". It fixes the typo for "use/user" and is gender neutral.